src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -22,6 +24,12 @@ def register_submission(
         config = ObjectsAPIConfig.get_solo()
         config.apply_defaults_to(options)
 
+        zgw_config = ZgwConfig.get_solo()
```

Comment:
Not sure if this should use the `ZGWConfig` or whether the DRC service, `informatieobjecttype` and `vertrouwelijkheidaanduiding` should be added to `ObjectsAPIConfig`.

Reply:
Let's separate them, you'd expect plugins and their configuration to be standalone, which currently isn't the case.

Reply:
changed in https://github.com/open-formulieren/open-forms/pull/501/commits/8ccd9d8f683680328b0ecad1d50ef2d99009c8ee